pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py

```diff
@@ -211,6 +221,37 @@ def verify_publish_is_skip(self, cfg, call_report):
         self.assertEqual(len(tasks), 1, tasks)
         self.assertEqual(tasks[0]['result']['result'], 'skipped', tasks[0])
 
+    def verify_files_in_dir(self, cfg, distributor_cfg):
+        """Verify the RPM rsync distributor has placed RPMs in the given path.
+
+        Verify that path ``{root}/{remote_units_path}/rpm`` exists in the
+        target system's filesystem, and that 32 RPMs are present in this
+        directory.
+
+        :param pulp_smash.config.ServerConfig cfg: Information about the system
+            onto which files have been published.
+        :param distributor_cfg: A dict of information about an RPM rsync
+            distributor.
+        :returns: Nothing.
+        """
+        # This method avoids calling command_string.split() to avoid issues
+        # with spaces and other funny character in path names.
+        cli_client = cli.Client(cfg)
+        sudo = () if utils.is_root(cfg) else ('sudo',)
+        path = distributor_cfg['config']['remote']['root']
+        remote_units_path = distributor_cfg['config'].get(
+            'remote_units_path',
+            'content/units',
+        )
+        for segment in _split_path(remote_units_path):
+            cmd = sudo + ('ls', '-1', path)
+            files = set(cli_client.run(cmd).stdout.strip().split('\n'))
+            self.assertIn(segment, files)
+            path = os.path.join(path, segment)
+        cmd = sudo + ('find', path, '-name', '*.rpm')
+        files = cli_client.run(cmd).stdout.strip().split('\n')
+        self.assertEqual(len(files), 32, files)
```

Comment:
We are using this constant on some places and this is directly linked to the fixtures repo. We may create a constant file and use that instead eventually so it will help updating if for some reason RPMs are added or removed from the fixtures repo.